WordPress/gutenberg
# Revert "Add defaultFontSizes option to theme.json (#56661)"

## What?

This reverts commit 940f0fe52cfd1e3599e507a7d3f83146914e978f.

Fixes #57889 by reverting https://github.com/WordPress/gutenberg/pull/56661


## Testing Instructions




Step-by-step reproduction instructions
- Open a homepage in the browser.
- Deactivate Gutenberg trunk
- Open the homepage in a new tab
- There should be no difference.
